src/OhadSoft.AzureLetsEncrypt.Renewal/Util/CertificateHelper.cs

```diff
@@ -50,7 +50,8 @@ public static async Task<IReadOnlyList<string>> GetLetsEncryptHostNames(IAzureWe
                 Trace.TraceInformation("Reading ARM certificate query response");
                 var body = await response.EnsureSuccessStatusCode().Content.ReadAsStringAsync().ConfigureAwait(false);
 
-                var letsEncryptCerts = ExtractCertificates(body).Where(s => s.Issuer.Contains(staging ? "Fake LE" : "Let's Encrypt"));
+                var issuerNames = staging ? new[] { "Fake LE" } : new[] { "Let's Encrypt", "R3", "R4", "E1", "E2" };
```

Comment:
Ideally these would be configurable but that would be a bigger change, so I can see this being good enough for now